Source/WebKit/UIProcess/Cocoa/WebPageProxyCocoa.mm

```diff
@@ -415,11 +415,11 @@ static bool exceedsRenderTreeSizeSizeThreshold(uint64_t thresholdSize, uint64_t
 {
 #if PLATFORM(IOS_FAMILY)
     auto iconAndSize = RenderThemeIOS::iconForAttachment(fileName, contentType, title);
-    auto icon = iconAndSize.icon;
-    size = iconAndSize.size;
 #else
-    auto icon = RenderThemeMac::iconForAttachment(fileName, contentType, title);
+    auto iconAndSize = RenderThemeMac::iconForAttachment(fileName, contentType, title);
 #endif
+    auto icon = iconAndSize.icon;
+    size = iconAndSize.size;
     return convertPlatformImageToBitmap(icon.get(), iconSize);
```

Comment:
Nit: Just `return convertPlatformImageToBitmap(iconAndSize.icon.get(), iconAndSize.size);`

Even better: Perhaps `convertPlatformImageToBitmap()` should accept a `RenderThemeCocoa::IconAndSize`, thought that might require making sure GTK, Windows, and Playstation have similar features.